examples/twittermap/web/public/javascripts/map/pointmap.js

```diff
@@ -266,7 +331,21 @@ angular.module('cloudberry.map')
               $scope.popUpTweet = L.popup({maxWidth:300, minWidth:300, maxHight:300});
               $scope.popUpTweet.setContent(tweetContent);
               $scope.currentMarker.bindPopup($scope.popUpTweet).openPopup();
+            });*/
```

Comment:
Yep. Thank you for pointing it out! It is because this version is not the final version of this task, so I just leave it there for the further use. The comment code is previously qiushi's code, and i am just not sure whether to delete it.